Assets/_Application/Samplers/PointerToWorldPosition.cs

```diff
@@ -1,14 +1,20 @@
 using UnityEngine.InputSystem;
 using UnityEngine;
 using System;
+using Netherlands3D.Twin.FloatingOrigin;
+using Netherlands3D.Coordinates;
 
 namespace Netherlands3D.Twin.Samplers
 {
     public class PointerToWorldPosition : MonoBehaviour
-    {
-        public Vector3 WorldPoint { get; private set; } //Constantly updated world point
+    {       
+        public Coordinate WorldPoint => worldPoint;
+
         private OpticalRaycaster opticalRaycaster;
         private Action<Vector3, bool> worldPointCallback;
+        private Coordinate worldPoint;
+        private int cullingMask = Physics.DefaultRaycastLayers;
```

Comment:
Ik denk dat de maskingLayers hier ook niet meegenomen moeten worden, dat heb ik al aangepast in `OpticalRaycaster.defaultRaycastLayers` dus ik denk niet dat je een culling mask hoeft op te geven hier. Anders zou ik die const gebruiken